pkg/infrastructure/services/messaging_test.go

```diff
@@ -80,12 +100,16 @@ func createServiceInstance(t *testing.T) v1alpha1.KogitoService {
 	}
 }
 
-func mockKogitoSvcReplies(t *testing.T, jsonResponse string) *httptest.Server {
-	handler := http.NewServeMux()
-	handler.HandleFunc(topicInfoPath, func(writer http.ResponseWriter, request *http.Request) {
-		_, err := writer.Write([]byte(jsonResponse))
-		assert.NoError(t, err)
-	})
+func mockKogitoSvcReplies(t *testing.T, handlers []serverHandler) *httptest.Server {
```

Comment:
I think it's time to move this to another go file, maybe `mock_svc_test.go` since it's serving different use cases.

Reply:
Done